registry/txt.go

```diff
@@ -35,6 +34,12 @@ const (
 	providerSpecificForceUpdate = "txt/force-update"
 )
 
+type endpointKey struct {
```

Comment:
Why not use `endpoint.EndpointKey` instead of redefining?

Reply:
There's no such thing. Did you mean `endpoint.Endpoint`?

Reply:
@Sewci0 it was moved recently https://github.com/kubernetes-sigs/external-dns/commit/4417ad48947f6f77b341cc00b340e93857465282.